torch/quantization/fx/_equalize.py

```diff
@@ -357,6 +366,40 @@ def scale_weight_node(
     scaled_bias = torch.mul(bias, next_equalization_scale)
     modules[node.target].bias = nn.Parameter(scaled_bias)
 
+def scale_weight_functional(node: Node, model: GraphModule, equalization_scale: torch.Tensor):
+    """ Scales the weight value for functional layers
+    """
+
+    # Find the next functional node so that we can construct the weight observer nodes
+    while node.op != 'output' and node.op != 'call_function':
+        node = node.next
```

Comment:
isn't the node here a weight eq observer? In that case, can't we find the user (similar to what we did for activation) to find the call function node?

Reply:
Yes, I forgot to update this part after the `prepare` changes, but I will do so